frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -161,6 +170,7 @@ const InspectionRow = ({ inspection, openDialog, setMissions, openScheduledDialo
                 {!isScheduled && (
                     <Button
                         variant="ghost_icon"
+                        disabled={isScheduleButtonDisabled}
```

Comment:
Suggest adding some kind of user feedback when button is disabled informing why it's disabled

Reply:
I can add a tooltip

Reply:
Or maybe some red text

Reply:
I have added tooltips and texts where convenient